compose/mpp/compilation-tests/build.gradle.kts

```diff
@@ -0,0 +1,143 @@
+        }
+        binaries.executable()
+    }
+    
+    @OptIn(ExperimentalWasmDsl::class)
+    wasmJs {
+        outputModuleName = "compilation-tests"
+        browser {
+            commonWebpackConfig {
+                outputFileName = "compilation-tests.js"
+                devServer = (devServer ?: KotlinWebpackConfig.DevServer()).apply {
+                    static = (static ?: mutableListOf()).apply {
+                        // Serve sources to debug inside browser
+                        add(project.rootDir.path)
+                        add(project.projectDir.path)
+                    }
+                }
+            }
+        }
+        binaries.executable()
+    }
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation(project(":compose:foundation:foundation"))
+                implementation(project(":compose:foundation:foundation-layout"))
+                implementation(project(":compose:material3:material3"))
+                implementation(project(":compose:material3:material3-window-size-class"))
+                implementation(project(":compose:material3:adaptive:adaptive"))
+                implementation(project(":compose:material3:adaptive:adaptive-layout"))
+                implementation(project(":compose:material3:adaptive:adaptive-navigation"))
+                implementation(project(":compose:material:material"))
+                implementation(project(":compose:mpp"))
+                implementation(project(":compose:runtime:runtime"))
+                implementation(project(":compose:ui:ui"))
+                implementation(project(":compose:ui:ui-graphics"))
+                implementation(project(":compose:ui:ui-text"))
+                implementation(project(":compose:ui:ui-backhandler"))
+                implementation(project(":lifecycle:lifecycle-common"))
+                implementation(project(":lifecycle:lifecycle-runtime"))
+                implementation(project(":lifecycle:lifecycle-runtime-compose"))
+                implementation(project(":navigation:navigation-common"))
+                implementation(project(":navigation:navigation-compose"))
+                implementation(project(":navigation:navigation-runtime"))
+                implementation(libs.kotlinStdlib)
+                implementation(libs.kotlinCoroutinesCore)
+
+                implementation("org.jetbrains.compose.material:material-icons-core:1.7.3") {
+                    // exclude dependencies, because they override local projects when we build 0.0.0-* version
+                    // (see https://repo1.maven.org/maven2/org/jetbrains/compose/material/material-icons-core-desktop/1.6.11/material-icons-core-desktop-1.6.11.module)
+                    exclude("org.jetbrains.compose.runtime")
+                    exclude("org.jetbrains.compose.ui")
+                }
+
+                implementation("org.jetbrains.compose.material3:material3:1.9.0-beta03")
```

Comment:
With this approach it won't really test any future changes

Reply:
removed

Reply:
After adding to this folder, we also need to add it here https://jetbrains.team/p/ui/repositories/compose-teamcity-config/files/main/.teamcity/compose/publish/subtasks/Task4ValidateTemplatesTutorials.kt